virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -47,6 +47,8 @@ Without an external monitoring system or a qualified human monitoring node healt
 ** Intel VT or AMD-V hardware virtualization extensions are enabled
 ** The no-execute (NX) flag is enabled
 
+* If FIPS mode is xref:../../installing/installing-fips.adoc#installing-fips[set up correctly at the OpenShift level], it is automatically enabled and no additional setup is needed for {VirtProductName}. FIPS must be enabled before your cluster's operating system boots for the first time.
```

Comment:
We use the {product-title} variable for instances of OpenShift (Container Platform)

s/at the OpenShift level/for your cluster

The phrase "it is automatically enabled" seems redundant 

Per ISG, use a prepositional phrase instead of an apostrophe to show the possessive form (cluster's)

Please see my suggested rewrite below.